index.js

```diff
@@ -31,6 +31,10 @@ const getPlatform = () => {
  * Installs Snapcraft on Linux
  */
 const runLinuxInstaller = () => {
+	if (process.env.INPUT_SKIP_INSTALL) {
```

Comment:
I think the check needs to be `process.env.INPUT_SKIP_INSTALL === "true"`. Otherwise, it will also match if the user passes in `"false"` (because the env variable is defined)

Reply:
I'm not sure. Does the yml bool `true` get passed in as a string `"true"`? What about also checking for `1`, `"1"` and `"True"`? Honestly I'd just prefer as-is (if defined in any way, assume true). Nobody should write `0` or `false`.

Reply:
Yeah, all environment variables are strings in Node.js.

IMO `false` should definitely be supported. In the docs of GitHub Actions, I've only seen `true` and `false` being used (i.e. nothing like `1` or `"True"`). I think I'd like to implement it the same way. Otherwise, it would probably be necessary to support [all possible boolean values in YAML](https://yaml.org/type/bool.html).

Reply:
I'll leave it as `=== "true"` for now :)